### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by grzesiek2010**
#### What has been done to verify that this works as intended?
I just reviewed the implemented changes.

#### Why is this the best possible solution? Were any other approaches considered?
It's a thing we agreed to do after each release.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
I think it can be merged without testing since as we know from our experience it's difficult to tell what should be tested so we can just test it on a daily basis.

#### Do we need any specific form for testing your changes? If so, please attach one.
No.

#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/opendatakit/docs/issues/new) and include the link below.
No.

#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [ ] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [ ] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/opendatakit/collect/blob/master/CONTRIBUTING.md#ui-components-style-guidelines)

### CODE REVIEW

**Review by seadowg — DISMISSED** [review]

File: collect_app/build.gradle [hunk 1]
```diff
@@ -209,22 +209,22 @@ dependencies {
...
 
     implementation("com.google.guava:guava:28.1-android")
 
-    implementation("com.google.code.gson:gson:2.8.5") {
+    implementation("com.google.code.gson:gson:2.8.6") {
         exclude group: 'org.apache.httpcomponents'
     }
 
-    implementation("com.google.http-client:google-http-client:1.31.0") {
+    implementation("com.google.http-client:google-http-client:1.32.1") {
```
Comment (seadowg) [inline collect_app/build.gradle]: I'm not sure what this is being used for? Just removed it and did a compile and it was all happy. Going to try and a full `checkAll` to see if we can remove it.

**Comment by seadowg** [comment]
@grzesiek2010 I agree that for this we should go ahead and merge without testing as it'd be hard without a full QA pass to define what exactly to test. I'd imagine sometimes we'll have updated deps that obviously prompt us to test something - a major release of mapbox for instance.

Leaving this unmerged for the moment until we have our high priority PR through.

File: collect_app/build.gradle (hunk 1 above)
Comment (seadowg) [reply collect_app/build.gradle]: Yeah looks like we can safely remove this and `com.google.oauth-client:google-oauth-client`

**Review by seadowg — CHANGES_REQUESTED** [review]
Looks like we're getting a fail at the `testDebug` level.

File: collect_app/build.gradle (hunk 1 above)
Comment (grzesiek2010) [reply collect_app/build.gradle]: I removed them

**Comment by grzesiek2010** [comment]
@seadowg You ere right about those unused libs, I have found two more so I think it's ready for final review and merging.

**Review by seadowg — COMMENTED** [review]
Did you give it a whirl through `./gradlew checkAll`?

**Comment by grzesiek2010** [comment]
> Did you give it a whirl through ./gradlew checkAll?

No because I have a new computer and a problem with running `checkTests` which I haven't fixed yet.

**Comment by grzesiek2010** [comment]
@seadowg I was able to solve the problem with `checkAll` locally and it seems fine so I think we can merge.

**Review by seadowg — APPROVED** [review]

**Pull request merged**